tests/spread/rockcraft/extension-flask-uncomment-blocks/task.yaml

```diff
@@ -0,0 +1,82 @@
+summary: flask extension test
+environment:
+  SCENARIO/bare: bare
+  SCENARIO/base_2204: ubuntu-22.04
+  BLOCK/bare: runtime-slices
+  BLOCK/base_2204: runtime-debs
+execute: |
+  NAME="flask-${SCENARIO//./-}"
+  ROCK_FILE="${NAME}_0.1_amd64.rock"
+  IMAGE="${NAME}:0.1"
+
+  run_rockcraft init --name flask-extension --profile flask-framework
+  sed -i "s/name: .*/name: ${NAME}/g" rockcraft.yaml
+  if [ "${SCENARIO}" = "bare" ]; then
```

Comment:
I wonder if this would be easier to read/maintain with regular `diff` and `patch`? As in, do the base `init`, update the project to use a bare-base, save that diff as `scenario-bare.patch`
It's a though for short-term improvements, we don't need to do this today

Reply:
I made this if/else to allow flexibility in the test. Supporting ubuntu@24.04 will be just a matter of adding a variant. Managing it with patches might become tricky